src/mctpd.c

```diff
@@ -2883,6 +3118,26 @@ static char* net_path(int net)
 	return buf;
 }
 
+static char* link_path(const char* link_name)
+{
+	size_t l;
+	char *buf = NULL;
+
+	l = strlen(MCTP_DBUS_PATH_LINKS) + 30;
+	buf = malloc(l);
+	if (!buf) {
+		return NULL;
+	}
+	/* can't use sd_bus_path_encode_many() since it escapes
+	   leading digits */
```

Comment:
will we have leading digits?

(I'm aware that this is copied from the peer_find function, but we have different naming semantics here)

Reply:
Interface names are allowed leading digits. eg `ip tuntap add name 123tun mode tun`